source/jormungandr/jormungandr/interfaces/v1/test/elastic_search_tests.py

```diff
@@ -504,7 +504,7 @@ def bragi_call_test():
         ]
     }
     mock_requests = MockRequests(
-        {'http://bob.com/autocomplete?q=rue+bobette&limit=10&timeout=2000': (bragi_response, 200)}
+        {'http://bob.com/autocomplete?limit=10&q=rue+bobette&timeout=2000': (bragi_response, 200)}
```

Comment:
just curious: why you switched `limit` and `q` ? does that matter?

Reply:
yes, as `MockRequest` is not a real api but does a dumb lookup in a dict the parameters must be sorted (since we compare the strings).